include/kep3/planet.hpp

```diff
@@ -130,21 +156,42 @@ struct planet_iface<Holder, T> : planet_iface<void, void>, tanuki::iface_impl_he
     KEP3_UDPLA_IMPLEMENT_GET(name, std::string, boost::core::demangle(typeid(T).name()))
     KEP3_UDPLA_IMPLEMENT_GET(extra_info, std::string, "")
 
-    [[nodiscard]] std::array<std::array<double, 3>, 2> eph(const epoch &ep) const final
+    [[nodiscard]] std::array<std::array<double, 3>, 2> eph(double mjd2000) const final
+    {
+        return this->value().eph(mjd2000);
+    }
+
+    [[nodiscard]] std::vector<double> eph_v(const std::vector<double> &mjd2000s) const final
     {
-        return this->value().eph(ep);
+        if constexpr (udpla_has_eph_v<T>) {
+            return this->value().eph_v(mjd2000s);
+        } else {
+            // We simply call a for loop.
+            auto size = mjd2000s.size();
+            std::vector<double> retval(size * 6u);
+            for (size_t i = 0u; i < mjd2000s.size(); ++i) {
```

Comment:
``size`` instead of ``mjd2000s.size()``. Also, ``i`` is of the wrong type (rare footage).